_posts/2021-02-08-infinispan-kubectl.adoc

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog
+title: Infinispan Kubectl
+permalink: /blog/:year/:month/:day/infinispan-kubectl
+date: '2021-02-08T12:00:00.000-00:00'
+author: remerson
+tags: [ "cli", "quarkus", "native", "kubernetes"]
+---
+
+Now that the Infinispan CLI is available as a TODO[native binary] it's possible to utilise it as a kubectl client plugin.
+
+Integrating with `kubectl` lets you install and remove the Infinispan Operator, create and delete Infinispan clusters, and obtain information about various resources.
+
+== Installation
+To install the CLI as a `kubectl` plugin, do the following:
+
+. Copy the `ispn-cli`, or create a hard link, to a file named "kubectl-infinispan".
+
+. Add `kubectl-infinispan` to your `PATH`.
+
+. Run `kubectl plugin list` to verify the CLI is added as a plugin.
+
+. Run `kubectl infinispan --help` to start using the CLI plugin.
+
+For more detailed instructions, please see the https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/#installing-kubectl-plugins[kubectl docs].
+
+== Examples
+Before executing the examples login to your Kubernetes cluster with `kubectl login`.
+
+==== Installing Infinispan Operator
+----
+kubectl infinispan install
+----
+
+==== Creating  Infinispan Clusters
+----
+kubectl infinispan create --replicas=2 myinfinispan
+kubectl infinispan get
+----
+
+==== Removing Clusters
```

Comment:
Is there an additional value between `kubectl infinispan delete myinfinispan` and `kubectl delete infinispan myinfinispan`?